lib/v8.js

```diff
@@ -368,7 +367,7 @@ class DefaultDeserializer extends Deserializer {
     }
     // Copy to an aligned buffer first.
     const buffer_copy = Buffer.allocUnsafe(byteLength);
-    copy(this.buffer, buffer_copy, 0, byteOffset, byteOffset + byteLength);
+    this.buffer.copy(buffer_copy, 0, byteOffset, byteOffset + byteLength);
```

Comment:
And now it's failing other tests, with `this.buffer.copy is not a function`. Is it forcing you to use the old `copy` function? Or should the tests be updated?

Reply:
https://nodejs.org/api/v8.html really makes no guarantees about what happens with malformed stuff, but probably a partial copy is better than memory corruption

Reply:
It seems that out of all the buffers ([Buffer](https://nodejs.org/api/buffer.html#class-buffer) | [TypedArray](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypedArray) | [DataView](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/DataView)) v8.deserialize can accept, only the node Buffer has a copy method. However, `Buffer.prototype.copy` currently doesn't care about this actually being a Buffer, so using `Buffer.copy.call` seems to work? I'm running tests locally right now

Reply:
`parallel/test-v8-serdes.js` and `parallel/test-child-process-advanced-serialization.js` start working again (the two which failed here). However, this feels like an extremely band-aid solution just to get the few checks from [`copyImpl` / `_copyActual`](https://github.com/nodejs/node/blob/98788dace632e64e7d7b13b10559b301bd390526/lib/buffer.js#L205)